infrastructure/src/main/kotlin/app/discord/repository/jpa/user/JpaUserEntityRepository.kt

```diff
@@ -0,0 +1,10 @@
+package app.discord.repository.jpa.user
+
+import app.discord.repository.jpa.user.schema.UserEntity
+import jakarta.persistence.Entity
+import org.springframework.data.jpa.repository.JpaRepository
+import java.util.*
+
+interface JpaUserEntityRepository: JpaRepository<UserEntity, UUID>{
```

Comment:
`UserEntity` 의 PK 타입은 UUID가 아니라, Long 타입입니다. 확인해주세요
```suggestion
interface JpaUserEntityRepository: JpaRepository<UserEntity, Long>{
```